editor/editor_log.cpp

```diff
@@ -123,21 +123,20 @@ void EditorLog::_load_state() {
 
 	Ref<ConfigFile> config;
 	config.instantiate();
-	Error err = config->load(EditorSettings::get_singleton()->get_project_settings_dir().plus_file("editor_layout.cfg"));
-
-	if (err == OK) {
```

Comment:
I just got rid of this if statement, and...

Reply:
Pro tip for reviewers: whitespace changes can be hidden with `?w=1` in the URL:
https://github.com/godotengine/godot/pull/49980/files?w=1